### ISSUE TITLE
Question: Calling non-variant sites

### ISSUE THREAD

**Issue by ja-Reeve** [luntergroup/octopus#249]
How can I get Octopus to output a VCF including non-variant sites?

**Comment by jelber2** [comment]
See https://luntergroup.github.io/octopus/docs/cli#--refcall

**Comment by ja-Reeve** [comment]
Thanks. I tried that before on a short 600kb region of the genome, but it only returns variant sites.

**My call:**
```
octopus \
                 -R $PATH/ref_genome.fasta \
                 -i $PATH/bam_list.txt \
                 -o $PATH/Octopus_trial1.6b.vcf.gz \
                 -T Contig38698:0-608273 \
                 --very-fast --refcall POSITIONAL
```

**Output (only showing 1st 10 calls & 3 samples):**
![Screen Shot 2022-11-04 at 16 51 41](https://user-images.githubusercontent.com/82411887/200022231-3c1453c6-2dde-4a4e-b932-dcaa90266483.png)

**Comment by jelber2** [comment]
Are you using the latest commit in the development branch?

**Comment by ja-Reeve** [comment]
Yes, version: 17a597d-20220708 is installed on the server I am using

**Comment by jelber2** [comment]
What if you remove --very-fast ?

**Comment by ja-Reeve** [comment]
I tried a test run, but it timed out after 5days. I will give it another go with more time.

**Comment by jelber2** [comment]
Maybe use samtools on a single sample to filter the bam file to just that region to test octopus without the --very-fast?

**Comment by ja-Reeve** [comment]
I tested this region with a single sample without `--very-fast``. I only returns indels and SNPs, no non-variant sites.
![Screen Shot 2022-11-04 at 19 17 56](https://user-images.githubusercontent.com/82411887/200047622-08bf9418-e9df-4112-8659-14a628b88793.png)

**Comment by jelber2** [comment]
Maybe I do not understand, but these are non-variant sites as well in the output you show. For example Contig38698:1 has phased genotype 0|0 . The remaining sites in the interval Contig38698:2-13763 are presumably not passing filter? So, you might need to keep the raw calls with `--keep-unfiltered-calls` and perhaps do some post processing to include the lines from the unfiltered file to the filtered file? I guess last time I used this, I realized I only wanted the confidently called sites, whether variant or non-variant.
